src/settings/MatchProfiles/MatchProfilesForm.js

```diff
@@ -53,18 +84,53 @@ export const MatchProfilesFormComponent = ({
     }
   };
 
-  const recordTypeId = FOLIO_RECORD_TYPES[initialValues.existingRecordType].captionId;
-  const recordTypeLabel = (
-    <FormattedMessage
-      id="ui-data-import.match.existing.record"
-      values={{ recordType: isEditMode ? <FormattedMessage id={recordTypeId} /> : '' }}
-    />
-  );
+  const onFieldSearch = (value, dataOptions) => {
+    return dataOptions.filter(o => new RegExp(`${value}`, 'i').test(o.label));
+  };
+
+  const onExistingRecordChange = record => {
+    const { type } = record;
```

Comment:
You don't use `record` in this function, type could be get from record in arguments above